spec/requests/04_put_route_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+RSpec.describe "PUT /ratingQuestions/:id" do
+  context "when the question exists" do
+    let!(:question) do
+      RatingQuestion.create!(title: "Hello World!")
+    end
```

Comment:
Put a space here between the `it` and the `let`.

Reply:
ok